### ISSUE TITLE
[v0.9.2] Random Chunks are removed when copying and pasting

### ISSUE THREAD

**Issue by jordanparki7** [Amulet-Team/Amulet-Map-Editor#650]
![image](https://user-images.githubusercontent.com/87543512/164796226-855b5d38-2711-47c1-abe6-9fa6fb6a7e4b.png)

As shown by the image above, when copying and pasting areas in v0.9.2 (in MCBE v1.18.30), some chunks just fail to paste. They appear to be pasted initially, but then after a second or so, they disappear.

Should you need any further information, I'm happy to test different scenarios etc.

Thanks :)

**Comment by jordanparki7** [comment]
This issue may appear to be similar to #645, but this issue is specifically when copying and pasting - I have not yet experienced #645.

**Comment by gentlegiantJGC** [comment]
Have you confirmed the selection?

**Comment by jordanparki7** [comment]
> Have you confirmed the selection?

Yes - the "preview" shows the chunks exist, but 1-2 seconds after confirming the selection, the chunk just disappears.

**Comment by Lantz5220** [comment]
Generated a new world and no matter how many times I've loaded the chunks area in-game, when I open it on the program, it still has some chunks that just won't load and it makes it hard for me to copy and past my builds because there will be missing chunks.

![Screenshot (61)](https://user-images.githubusercontent.com/97374405/164977006-fc535294-6dfe-4af1-bada-0e4df855d4c4.png)

**Comment by gentlegiantJGC** [comment]
@Lantz5220 this is a different issue. Please do not hijack other issues. It look like in the latest release the game is more strict on which chunks get saved. The black chunks are simply chunks the game did not save and as such are not there for Amulet to display.

**Comment by gentlegiantJGC** [comment]
To confirm the original issue @jordanparki7 
You have copied some islands from the world.
You then paste them somewhere else in the world and click confirm to write it to the world.
Are the chunks you pasted into blue or black?
Once you click back to the select tool the pasted region is partially there or not there at all?

Could you send some more screenshots of the whole process so I can better understand

**Comment by jordanparki7** [comment]
Correct - I copied some islands from the world, then deleted them from the world. I then pasted the islands in another area (clicking confirm). I pasted the islands into unloaded (black) chunks, but I have had the same issue with loaded (blue) chunks, as that is what I originally thought the issue was. After I click back to the select tool, the pasted region is all there, then after about 2 seconds, a couple of chunks from the pasted region disappears.

Yes - I will try and send some more screenshots with exact steps I take later :)

**Comment by gentlegiantJGC** [comment]
Does this happen every time or only occasionally?
Are there any errors in the console when this happens?

**Comment by gentlegiantJGC** [comment]
If this happens again can you try flying away from the area and then fly back.
I am 99% sure that the actual terrain is still there but it has not generated the mesh.

**Comment by gentlegiantJGC** [comment]
You can also confirm this by pressing alt to inspect the block in the area

**Comment by jordanparki7** [comment]
Nope - the actual terrain does not exist when you enter the world - alt inspect does not work on these blocks.

**Comment by gentlegiantJGC** [comment]
And are there any errors in the console when this happens?

**Comment by gentlegiantJGC** [comment]
@stealthyexpertx do you have any more details on this?

**Comment by StealthyExpertX** [comment]
@gentlegiantJGC Sorry for the late response didn't see this message, I have not personally run into this but have heard a few people complain about it in the Amulet discord server.
